execution/gethexec/espresso_sequencer.go

```diff
@@ -73,11 +73,6 @@ func (s *EspressoSequencer) createBlock(ctx context.Context) (returnValue bool)
 
 	}
 
-	if len(arbTxns.Transactions) == 0 {
```

Comment:
Removing this because we still need to justify empty blocks

Reply:
What does "Justify empty blocks" mean? I think this code is to prevent L2 from generating "empty blocks".